### DISCUSSION TITLE
Peptiderive Validation failed:  complex.pdb must be specified

### DISCUSSION THREAD

**Discussion by MertYenen** [RosettaCommons/rosetta#182] [ROSIE]
Hello everyone,
I have submitted a cluspro docked protein in peptiderive but it says to me:

"Submitting Peptiderive job failed!

Validation failed as:
the complex.pdb must be specified

Please correct this errors and re-submit!"

I don't know the exact problem(it probably tells me to extract alternate locations), I have tried to fix pdb in FoldX with the repair pdb manual but ı don't know what to do further

**Comment by roccomoretti** [comment]
On the [Peptiderive submission page](https://rosie.rosettacommons.org/peptiderive/submit), near the top there's a line with "Input PDB file ... or enter a PDB ID to be fetched from RCSB ..."  the error message you're getting indicates that those entries weren't properly filled out when you attempted to submit.

One issue may be that uploading a file locally is a two-step process. First you need to "Browse" to find the file on your local machine, but then you need to hit the "Upload" button to actually send the file to the server. Only when you get the little image of your protein on the submission page with the "Show contents" button is the file successfully uploaded.

**Reply by MertYenen** [reply]
I am such an idiot:(( Even though I have done peptiderive jobs before I totally forget to press upload. Thank you for your detailed explanation
